internal/jimm/juju/jimm.go

```diff
@@ -328,3 +329,51 @@ func (j *JujuManager) PrepareModelMigration(
 
 	return migrationToken, nil
 }
+
+// ListMigrationTargets returns a list of controllers the model could be migrated to
```

Comment:
also mention that it returns controller of a higher version. and end the godoc with a full stop.